src/translations/screens/subscreens/TripDetails.ts

```diff
@@ -11,7 +11,11 @@ const TripDetailsTexts = {
           `Steg ${stepNumber}, ${travelMode}`,
           `Step ${stepNumber}, ${travelMode}`,
         ),
-
+      lastPassedStop: (stopPlaceName: string, time: string) =>
+        _(
+          `Passerte ${stopPlaceName} kl. ${time}`,
+          `Passed ${stopPlaceName} at ${time}`,
+        ),
```

Comment:
Should be more precise regarding `StopPlace` or `Quay`. Here the parameter is named `stopPlaceName`, but you are sending in `quay.name`.